funcx_endpoint/funcx_endpoint/endpoint/interchange.py

```diff
@@ -210,6 +210,11 @@ def load_config(self):
         self.executors = {}
         for executor in self.config.executors:
             logger.info(f"Initializing executor: {executor.label}")
+            if not executor.endpoint_id:
```

Comment:
Can you explain why these need to match? Eventually, we'll have multiple executors attached to each endpoint where each would have its own unique ID.

Reply:
The original issue post mentioned this mismatch. In the current implementation (without this PR), when an executor is launched, a random string would be assigned to its `endpoint_id` attribute. This PR makes the executor's `endpoint_id` the same as the endpoint's. 

I was thinking do we need an additional `executor_id` attribute for an executor, as a unique identifier?

Reply:
Okay, I understand what you are doing here. Let's address the multi-executor situation when we get there.